discussions/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
This file will probably not exist if we choose to make a tests folder which I think we are going to.